src/imgui.ini

```diff
@@ -0,0 +1,5 @@
+[Window][Debug##Default]
+Pos=60,60
+Size=400,400
+Collapsed=0
+
```

Comment:
You keep accidentally adding this. Might I suggest adding this line to the gitignore

Reply:
Just add this to .gitignore
```
imgui.ini
```